packages/babylon/src/tokenizer/index.js

```diff
@@ -4,6 +4,7 @@
 
 import type { Options } from "../options";
 import type { Position } from "../util/location";
+import charCodes from "../util/charCodes";
```

Comment:
This should be `charCodes, { isDigit }`?

Guess once we make them individual exports we can just do `* as charCodes`

Reply:
Yes, I mentally planned to use the * import.